kong/templates/deploy-kong.yaml

```diff
@@ -16,34 +16,15 @@ spec:
       labels:
         app: {{ .Values.Component }}
     spec:
-      {{- if eq .Values.IstioSidecar.enabled true }}
-      initContainers:
-      - args:
-        - -p
-        - "15001"
-        - -u
-        - "1337"
-        - -m
-        - REDIRECT
-        - -i
-        - 10.16.0.0/14
-        - -x
-        - ""
-        - -d
-        - ""
-        image: {{ .Values.IstioSidecar.hub }}/proxy_init:{{ .Values.IstioSidecar.tag }}
-        imagePullPolicy: IfNotPresent
-        name: istio-init
-        resources: {}
-        securityContext:
-          capabilities:
-            add:
-            - NET_ADMIN
-          privileged: true
+      {{- if (not (empty .Values.ImagePullSecrets)) }}
+      imagePullSecrets:
+      {{- range .Values.ImagePullSecrets }}
+         - name: {{ . }}
```

Comment:
Judging by the template output, the spacing here is still wrong.. I count three spaces where there should be either 2 or zero